### ISSUE TITLE
RadarSensorView Multiple Rx Channels

### ISSUE THREAD

**Issue by kmeids** [OpenSimulationInterface/open-simulation-interface#454]
#### Describe the problem
Considering one radar with multiple Rx antennas, it seems that under  **osi3::RadarSensorView**/**osi3::RadarSensorViewReflection** messages there is no way to separately add/encode the radar returns per channel.
One can add multiple reflections (returns) however not multiple reflections/channel.

Here is a screenshot that illustrates the problem:

![image](https://user-images.githubusercontent.com/25367407/102070448-7000fd80-3dff-11eb-89b9-be4e2f0ff83d.png)

#### Ask your question
So my basic question is, is there something that I am missing or is this currently not supported in which case an extension would be required.

**Comment by pmai** [comment]
The way that OSI currently would model multiple rx antennas would be via multiple RadardSensorViewConfigurations&RadarSensorViews: Each of those has its own physical sensor id (which is distinct from the virtual sensor id of the overall SensorViewConfiguration/SensorView), allowing clear identification of the data. For multiple Rx antennas this is also fairly efficient, since rays have to be started for each Rx antenna anyway.

**Comment by kmeids** [comment]
> The way that OSI currently would model multiple rx antennas would be via multiple RadardSensorViewConfigurations&RadarSensorViews: Each of those has its own physical sensor id (which is distinct from the virtual sensor id of the overall SensorViewConfiguration/SensorView), allowing clear identification of the data. 

Yes but [RadarSensorView](https://opensimulationinterface.github.io/open-simulation-interface/structosi3_1_1RadarSensorView.html) does not have an **id**  field. Or am I missing something here?

**Comment by thomassedlmayer** [comment]
Closing this PR due to inactivity. Feel free to reopen or create a new one if you'd like to continue the discussion or address the feedback.
